recipes/cider

```diff
@@ -1,4 +1,7 @@
 (cider :fetcher github
        :repo "clojure-emacs/cider"
-       :files ("*.el" "clojure.sh" "lein.sh" (:exclude ".dir-locals.el"))
+       :files (;; new code layout:
+               "lisp/*.el" "script/*.sh"
```

Comment:
cc/ @bbatsov

Reply:
Shouldn't we name `script` `bin` or `exe`? I think that's a more common naming convention. (or at least it's quite popular in Ruby land, where I mostly work :D )

Reply:
IDK, both choices kind of suggest that the contents are binaries

Reply:
More like executable files. That's popular in `Clojure-land` as well - see https://github.com/technomancy/leiningen/tree/github/bin